crates/bitwarden-uniffi/src/auth/mod.rs

```diff
@@ -110,10 +110,13 @@ impl AuthClient {
         password: String,
         encrypted_user_key: String,
     ) -> Result<String> {
+        let encrypted_user_key = encrypted_user_key
+            .parse::<EncString>()
```

Comment:
We can also parse into `WrappedSymmetricKey` here and remove the `.into()`
```suggestion
            .parse::<WrappedSymmetricKey>()
```